core/new-gui/src/app/workspace/component/navigation/navigation.component.scss

```diff
@@ -7,10 +7,17 @@
   z-index: 1;
 }
 
+
+.user-icon-navigation-page{
+  position:relative;
+  top:-3px;
+  left:890px;
+}
+
 .texera-navigation-padded {
   height: 100%;
   width: 100%;
-  overflow: hidden;
+  overflow: visible; //was hidden previously
```

Comment:
this overflow: visible should not be applied to entire `.texera-navigation-padded`. add it to the login button itself.